inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/auth/tenant/TenantAuthenticatingFilter.java

```diff
@@ -48,23 +55,24 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
         HttpServletRequest httpServletRequest = (HttpServletRequest) request;
         Subject subject = SecurityUtils.getSubject();
 
+        UserInfo loginUserInfo;
         // pre-check
         if (!subject.isAuthenticated()) {
-            log.error("The request should be authed before tenant authentication, user:{}, path:{}",
-                    subject.getPrincipal(),
-                    httpServletRequest.getServletPath());
-            ((HttpServletResponse) response).sendError(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
-            return;
+            log.debug("the request is not authed before tenant authentication, use default user:{}, path:{}",
+                    DEFAULT_USER, httpServletRequest.getServletPath());
+            loginUserInfo = defaultUserInfo();
```

Comment:
Will it exist that when authentication is enabled, the user is not logged, and finally the default user info is set here by mistake?

Reply:
I think it's only necessary to set a default user when OpenAPI authentication is not enabled.

Reply:
The idea of @fuweng11 is a good solution.